src/Umbraco.Core/Models/MediaExtensions.cs

```diff
@@ -15,7 +16,8 @@ public static class MediaExtensions
         /// </summary>
         public static string GetUrl(this IMedia media, string propertyAlias, ILogger logger)
         {
-            var propertyType = media.PropertyTypes.FirstOrDefault(x => x.Alias.InvariantEquals(propertyAlias));
+            var contentType = Current.Services.MediaTypeService.Get(media.ContentTypeId);
```

Comment:
wait what? All of this just to check if the alias actually exists? Shouldn't we already know this based on the `media.Properties` collection?